apps/meeting/package-lock.json

```diff
@@ -14,8 +14,8 @@
         "@types/webpack-env": "^1.15.1",
         "@typescript-eslint/eslint-plugin": "^2.19.2",
         "@typescript-eslint/parser": "^2.19.2",
-        "amazon-chime-sdk-component-library-react": "^2.11.1",
-        "amazon-chime-sdk-js": "^2.20.0",
+        "amazon-chime-sdk-component-library-react": "/Volumes/workplace/github/react/amazon-chime-sdk-component-library-react/amazon-chime-sdk-component-library-react-2.11.1.tgz",
+        "amazon-chime-sdk-js": "^2.21.1",
```

Comment:
we need to make sure to update this again before this pr is merged. Just a reminder.